docs/releases/v2.2.1/sdlc-compliance.md

```diff
@@ -0,0 +1,48 @@
+SSDLC Compliance Report: Atlas Kubernetes Operator Manager v2.2.1
+=================================================================
+
+- Release Creators: roo.thorp@mongodb.com, jose.vazquez@mongodb.com
+- Created On:      2024-04-18
+
+Overview:
+
+- **Product and Release Name**
+
+    - Atlas Kubernetes Operator v2.2.1, 2024-04-18. 
+    - Release Type: Minor
+
+- **Process Document**
+  - http://go/how-we-develop-software-doc
+
+- **Tool used to track third party vulnerabilities**
+  - Silk
+
+- **Dependency Information**
+  - See SBOMS (CycloneDX in JSON format) for [Intel](./linux-amd64.sbom.json) or [ARM](./linux-arm64.sbom.json)
```

Comment:
Probably worth adding a note here that those are SBOM Lites (not the augmented ones).